src/cpp/session/modules/SessionSource.R

```diff
@@ -184,8 +184,10 @@
    
    if (length(args) > 0)
    {
-      for (ee in args)
-         freeVars <- c(freeVars, codetools:::walkCode(ee, w))
+      for (i in seq_along(args))
+         # Filter empty string https://github.com/rstudio/rstudio/issues/5285
+         if (!identical(args[[i]], expression()))
```

Comment:
I'm realizing now that this line is unnecessary as it's legal to access an R missing value that lives within a list object. For example:

```
> x <- quote(expr = )
> x
Error: argument "x" is missing, with no default
> l <- list(x = quote(expr = ))
> l$x

```

So simply re-structuring the loop so that we access `args[[i]]` through an index rather than looping fixes the issue.

(Also note: it really is `quote(expr = )` as opposed to an empty `expression()` object)